### PR TITLE
LGA-894 Error Messages

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
## What does this pull request do?

Converted error messages to new style.

## Any other changes that would benefit highlighting?

This appears to also fix 922

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/templates/contact.html [hunk 1]
```diff
@@ -154,26 +154,23 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
...
 
   <script type="text/html" id="postcodeNotEnteredText">
-    <div class="form-row field-error">
-      <p>
-        {{ _('Must contain a valid postcode') }}
-      </p>
-    </div>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">{% trans %}Error{% endtrans %}:</span>
+      <span class="cla-error-message">{{ _('Must contain a valid postcode') }}</span>
```
Comment (said-moj) [inline cla_public/templates/contact.html]: Why the switch between {% trans %} and {{ _('') }}

File: cla_public/templates/contact.html [hunk 2]
```diff
@@ -154,26 +154,23 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
...
 
   <script type="text/html" id="noAddressesFoundText">
-    <div class="form-row field-error">
-      <p>
-        {{ _('No addresses were found with that postcode, but you can still enter your address manually') }}
-      </p>
-    </div>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">{% trans %}Error{% endtrans %}:</span>
+      <span class="cla-error-message">{{ _('No addresses were found with that postcode, but you can still enter your address manually') }}</span>
```
Comment (said-moj) [inline cla_public/templates/contact.html]: Why the switch between {% trans %} and {{ _('') }}

File: cla_public/templates/contact.html [hunk 3]
```diff
@@ -154,26 +154,23 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
...
 
   <script type="text/html" id="requestFailedText">
-    <div class="form-row field-error">
-      <p>
-        {{ _('Request failed: ') }} <%= textStatus %>, <%= error %>
-      </p>
-    </div>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">{% trans %}Error{% endtrans %}:</span>
+      <span class="cla-error-message">{{ _('Request failed: ') }} <%= textStatus %>, <%= error %></span>
```
Comment (said-moj) [inline cla_public/templates/contact.html]: Why the switch between {% trans %} and {{ _('') }}

File: cla_public/templates/contact.html (hunk 2 above)
Comment (MalcolmVonMoJ) [reply cla_public/templates/contact.html]: No reason.  The {{ _('') }} approach was already there - I just added stuff around it.  
The {% trans %} approach was copied from elsewhere.

File: cla_public/templates/contact.html (hunk 3 above)
Comment (MalcolmVonMoJ) [reply cla_public/templates/contact.html]: No reason.  The {{ _('') }} approach was already there - I just added stuff around it.  
The {% trans %} approach was copied from elsewhere.

File: cla_public/templates/contact.html (hunk 1 above)
Comment (MalcolmVonMoJ) [reply cla_public/templates/contact.html]: No reason.  The {{ _('') }} approach was already there - I just added stuff around it.  
The {% trans %} approach was copied from elsewhere.

Comment (said-moj) [reply cla_public/templates/contact.html]: In that case, can we make them consistent for the parts we are updating, so maybe replace the  {{ _('') }} with trans

File: cla_public/templates/contact.html (hunk 3 above)
Comment (MalcolmVonMoJ) [reply cla_public/templates/contact.html]: Changed to `{{ _('') }}`

File: cla_public/templates/contact.html (hunk 2 above)
Comment (MalcolmVonMoJ) [reply cla_public/templates/contact.html]: Changed to `{{ _('') }}`

File: cla_public/templates/contact.html (hunk 1 above)
Comment (MalcolmVonMoJ) [reply cla_public/templates/contact.html]: Changed to `{{ _('') }}`

**Pull request merged**